README.md

```diff
@@ -1 +1,5 @@
-# python-f3d-extras
\ No newline at end of file
+# python-f3d-extras
+
+The `f3d_extras` module provides utility functions and additional functionality on top of [`f3d`](https://f3d.app)'s [Python bindings](https://pypi.org/project/f3d/).
+
+Example of its use can be found in the [`examples/`](https://github.com/f3d-app/python-f3d-extras/blob/main/examples) directory of the source repository.
```

Comment:
Add a how to use ?

you mentionned the `pip install url` stuff, that would be nice ?

Also please mention the minimum version of F3D required and how to install it (eg `pip install`)